library/func_misc.py

```diff
@@ -401,3 +421,51 @@ def Call(self, args):
                 return value.Dict(d)
 
         raise error.TypeErr(val, 'Dict() expected List or Dict', loc.Missing)
+
+
+class Split(vm._Callable):
+
+    def __init__(self, splitter):
+        # type: (split.SplitContext) -> None
+        vm._Callable.__init__(self)
+        self.splitter = splitter
+
+    def Call(self, args):
+        # type: (typed_args.Reader) -> value_t
+        s = args.PosStr()
+
+        ifs = None  # type: str
+        if args.NumPos():
+            ifs = args.PosStr()
+
+        args.Done()
+
+        # mycpp: rewrite list comprehension over sum type
+        l = []  # type: List[value_t]
+        for elem in self.splitter.SplitForWordEval(s, ifs=ifs):
+            l.append(value.Str(elem))
+
+        return value.List(l)
+
+
+class Glob(vm._Callable):
+
+    def __init__(self, globber):
+        # type: (glob_.Globber) -> None
+        vm._Callable.__init__(self)
+        self.globber = globber
```

Comment:
Arguably we can put a Call() method directly on splitter and globber, but it doesn't matter much either way

Reply:
Ah yeah, I thought about doing that, but it felt cleaner for the builtin code to be closer to the other builtins